eq-author/src/App/page/Design/CalculatedSummaryPageEditor/AnswerSelector.js

```diff
@@ -0,0 +1,475 @@
+  padding: 0.5em 1em;
+`;
+
+const SuggestionsTitle = styled.h3`
+  font-size: 0.8em;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+  margin: 0;
+  color: #7a7a7a;
+`;
+
+const SuggestionsList = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+`;
+
+const SuggestionsListItem = styled.li`
+  margin: 0;
+  padding: 0;
+`;
+
+export const SuggestionButton = styled.button`
+  padding: 0.25em 1em;
+  background: transparent;
+  appearance: none;
+  font-size: 1em;
+  border: none;
+  text-align: left;
+  cursor: pointer;
+  width: 100%;
+  color: #4a4a4a;
+  display: flex;
+  align-items: center;
+
+  &:hover {
+    background: ${colors.lighterGrey};
+  }
+
+  &:focus {
+    outline: none;
+    box-shadow: 0 0 0 3px ${colors.tertiary};
+  }
+`;
+
+const SuggestionText = styled.span`
+  flex: 1 1 auto;
+`;
+
+const SuggestionTitle = styled.div`
+  font-weight: bold;
+`;
+
+const SuggestionSubtitle = styled.div`
+  font-size: 0.9em;
+`;
+
+const SuggestionAnswers = styled.div`
+  font-size: 0.8em;
+  font-weight: bold;
+  color: #7a7a7a;
+  letter-spacing: 0.05em;
+  margin-left: auto;
+  flex: 0 0 auto;
+  text-transform: uppercase;
+`;
+
+const TextButton = styled.button`
+  appearance: none;
+  border: none;
+  font-size: 0.85rem;
+  letter-spacing: 0.05rem;
+  margin: 0 0.5em;
+  color: ${colors.primary};
+  background: transparent;
+  font-weight: bold;
+  text-transform: uppercase;
+
+  &:focus {
+    outline: none;
+    border-color: ${colors.blue};
+    outline-color: ${colors.blue};
+    box-shadow: 0 0 0 3px ${colors.tertiary};
+  }
+`;
+
+const Answers = styled.div`
+  padding: 1em;
+`;
+
+const SectionList = styled.ul`
+  list-style: none;
+  margin: 0 0 0.5em;
+  padding: 0;
+`;
+
+const SectionListItem = styled.li`
+  margin: 0;
+`;
+
+const SectionHeader = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+  margin-bottom: 0.7em;
+`;
+
+const RemoveAllButton = styled(TextButton)`
+  border: none;
+  font-size: 0.8rem;
+  margin: 0 0 0 auto;
+`;
+
+const SectionTitle = styled.div`
+  font-weight: bold;
+  font-size: 0.9em;
+  color: #807d77;
+`;
+
+const AnswerList = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-flow: row wrap;
+`;
+
+const AnswerListItem = styled.li`
+  margin: 0 0 0.5em;
+  width: 100%;
+`;
+
+const SelectButton = styled(Button)`
+  width: 100%;
+`;
+
+const Empty = styled.div`
+  color: #7a7a7a;
+  text-align: center;
+  padding: 1em 2em 2em;
+
+  &::before {
+    display: block;
+    content: url(${iconInfo});
+  }
+`;
+
+const EmptyTitle = styled.h3`
+  font-size: 1em;
+  margin: 0 0 0.5em;
+`;
+
+const EmptyButton = styled(Button)`
+  padding: 0.5em 1em;
+`;
+
+const EmptyText = styled.div`
+  font-size: 0.9em;
+  margin-bottom: 1em;
+`;
+
+const NoShortcuts = styled.div`
+  padding: 0.5em 1em;
+  font-weight: normal;
+  font-size: 0.9em;
+`;
+
+const translateSelectedAnswers = selectedAnswers => {
```

Comment:
As you mentioned. These answers can no longer go across multiple sections so this isn't needed anymore.